components/broadcast_talent_table/commons/broadcast_talent_table.lua

```diff
@@ -303,15 +305,15 @@ end
 function BroadcastTalentTable:_tierDisplay(tournament)
 	-- `tournament.extradata` is not the extradata of the tournament but of the broadcaster (they got merged together)
 	if Logic.readBool((tournament.extradata or {}).showmatch) then
-		return 'Showmatch'
+		return 'Showmatch', 'B1'
```

Comment:
is b1 the best choice here?

Reply:
but some wikis might have other sorting values for showmatch

maybe checking for that might be a good idea (am on phone for rest of today)

Reply:
This should be legacy behavior anyway, so IMO this could be moved further down to be a fallback iff tier/tiertype aren't properly set.